backend/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM call-home-shared
```

Comment:
Is this name documented anywhere other than docker compose? If not, then this dockerfile isn't usable outside of docker compose. Might be worth creating an npm script in shared/ called `dockerize` that builds and tags the image accordingly.

Reply:
good idea, will do that